composer.json

```diff
@@ -86,7 +86,7 @@
         "spryker-feature/product-rating-reviews": "^202212.0",
         "spryker-feature/product-relations": "^202212.0",
         "spryker-feature/product-sets": "^202212.0",
-        "spryker-feature/promotions-discounts": "^202212.0",
```

Comment:
spryker/discounts is also a part of https://release.spryker.com/features/view/116 feature. Do we need to update it as well? (for all PRs)

Reply:
@artem-png, this package is already updated.